common/djangoapps/enrollment/views.py

```diff
@@ -76,7 +76,7 @@ class EnrollmentUserThrottle(UserRateThrottle, ApiKeyPermissionMixIn):
     """Limit the number of requests users can make to the enrollment API."""
     THROTTLE_RATES = {
         'user': '40/minute',
-        'staff': '1200/minute',
+        'staff': '1800/minute',
```

Comment:
1. Can you show splunk or other data to show what this needs to be to avoid the problem for now?
2. Did we review what is a reasonable rate limit that would not cause a problem for lms?  Maybe it could be much higher and we don't have to bump it little by little each time?
3. We should discuss to determine if there is a separate long-term solution.

Reply:
@robrap The number 1800 is simply around 150 above what I am seeing in Splunk in my current research.  I am still reaching out to stakeholders before merging for input on what is appropriate here and if the throttle is needed.

Reply:
+1 can you link to the errors in splunk or newrelic that this is expected to resolve,  I don't see any http failures on the ecommerce side.

Reply:
@robrap Let me know if you still have these questions after our discussion.

Reply:
@staubina: It wouldn't hurt to add urls to the PR as well.

Reply:
I have added the URLs to the Jira ticket to centralize the data.  Also adding the ticket url to the PR.

Reply:
@feanil Is the data in the Jira ticket enough information?